sdk/src/main/scala/com/horizen/account/api/http/AccountTransactionApiRoute.scala

```diff
@@ -735,18 +735,11 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
   private def checkKeyRotationProofValidity(body: ReqCreateKeyRotationTransaction): Unit = {
     val index = body.keyIndex
     val keyType = body.keyType
-    val newKey = SchnorrPropositionSerializer.getSerializer.parseBytes(BytesUtils.fromHexString(body.newKey))
-    val newKeySignature = SchnorrSignatureSerializer.getSerializer.parseBytes(BytesUtils.fromHexString(body.newKeySignature))
     if (index < 0 || index >= params.signersPublicKeys.length)
       throw new IllegalArgumentException(s"Key rotation proof - key index out for range: $index")
 
     if (keyType < 0 || keyType >= KeyRotationProofTypes.maxId)
       throw new IllegalArgumentException("Key type enumeration value should be valid!")
-
-    val messageToSign: Array[Byte] = newKey.getHash
```

Comment:
I removed it can I remember we added this part "just because we can". All the other signatures and this one are checked later in message processor. Now as we can't do it right away, and I voted for simplifying the code here, because this check will be done later in the process

Reply:
ok, fine to me